### DISCUSSION TITLE
[Feature] Ability to favorite or pin an album

### DISCUSSION THREAD

**Discussion by NYR10146** [immich-app/immich#13356] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

As someone with a lot of albums, I would love the ability to favorite an album, similarly to how we can already do so with photos. 

Alternatively, an option to Pin or Star an album so it stays at the top of my Albums list would be good. This would allow users to easily access commonly used or important albums without relying on changing the sort order.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2321
